packages/next/pages/_document.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { Component, ReactElement, ReactNode, useContext } from 'react'
-import flush from 'styled-jsx/server'
+import { useStyleRegistry } from 'styled-jsx'
```

Comment:
Does this mean the new API is bundled together into the main package so we don’t have code splitting?

Reply:
I think modern webpack can tree-shake it out? the subpath import `/server` is designed while webpack 1 or 2 is on active service. 
Now both client and sever relies on the same react context. Previously `flush` is still relies on the registry code

Reply:
Makes sense 👍

> Now both client and sever relies on the same react context. Previously flush is still relies on the registry code